From 16d1217f8a552a20ecbeb1490e1f054a97f4d03e Mon Sep 17 00:00:00 2001
From: Francois Choleau <francois.choleau@st.com>
Date: Fri, 13 Sep 2024 10:09:15 -0400
Subject: [PATCH 685/841] feat(dt-bindings): add declaration of RIF_UNUSED

RIF_UNUSED was missing

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git ccc4ef8a098f8eb43552f846b5ad5f11fc03d422 feat(dt-bindings): add declaration of RIF_UNUSED]
Signed-off-by: Francois Choleau <francois.choleau@st.com>
Change-Id: Ic3211f6f3450ce8aaf7d4015db9ee56fababc8ae
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/404349
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 include/dt-bindings/soc/rif.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/dt-bindings/soc/rif.h b/include/dt-bindings/soc/rif.h
index 7ded90d9a..a2f89568e 100644
--- a/include/dt-bindings/soc/rif.h
+++ b/include/dt-bindings/soc/rif.h
@@ -49,6 +49,9 @@
 #define RIF_UNLOCK		0x0
 #define RIF_LOCK		0x1
 
+/* Used when a field in a macro has no impact */
+#define RIF_UNUSED		0x0
+
 #define RIFPROT(rifid, sem_list, sec, priv, scid, sem_en, cfen) \
 	(((rifid) << 24) | ((sem_list) << 16) | ((priv) << 9) | ((sec) << 8) | ((scid) << 4) | \
 	 ((sem_en) << 1) | (cfen))
-- 
2.47.2

